### ISSUE TITLE
MOSIP-34370: PMP UI-Observed access violation when "IBM Equal Access Accessibility Checker" executed on PMP UI

### ISSUE THREAD

**Issue by Ivanmeneges** [mosip/partner-management-services#1832]
## Steps to Reproduce:

1. Add extension "IBM Equal Access Accessibility Checker" extension to the browser

1. Right click on the browser and select inspect mode

1. Select Accessibility Assessment from the menu option

1. Click on scan button and observe the failures

*Scenario 2*: 

Observed the no of isssues on scanning IBM Equal Access Accessibility Checker for the below tabs

75 issues found - Home page
104 issues found - Cert trust store
116 issues found - Partners
106 issues found - Partner policy mapping
147 issues found - SBI device
131 issues found - Device Tab
105 issues found- FTM chip
125 issues found - OIDC tab
103 issues found- API key tab

*Actual Result*: There are accessibility violations observed while running the "IBM Equal Access Accessibility Checker" tool.

*Expected Result*: There should not be any access violations on the pages

[Accessibility_Report-Partner Management (1).xlsx](https://mosip.atlassian.net/browse/MOSIP-34370)
[Accessibility_Report-Partner Management (2).xlsx](https://mosip.atlassian.net/browse/MOSIP-34370)
[Accessibility_Report-Partner Management (3).xlsx](https://mosip.atlassian.net/browse/MOSIP-34370)
[Accessibility_Report-Partner Management (4).xlsx](https://mosip.atlassian.net/browse/MOSIP-34370)
[Accessibility_Report-Partner Management (5).xlsx](https://mosip.atlassian.net/browse/MOSIP-34370)

## 📎 Attachments

- [Accessibility_Report-Partner Management (1).xlsx](https://mosip.atlassian.net/browse/MOSIP-34370)

- [Accessibility_Report-Partner Management (10).xlsx](https://mosip.atlassian.net/browse/MOSIP-34370)

- [Accessibility_Report-Partner Management (11).xlsx](https://mosip.atlassian.net/browse/MOSIP-34370)

- [Accessibility_Report-Partner Management (12).xlsx](https://mosip.atlassian.net/browse/MOSIP-34370)

- [Accessibility_Report-Partner Management (13).xlsx](https://mosip.atlassian.net/browse/MOSIP-34370)

- [Accessibility_Report-Partner Management (14).xlsx](https://mosip.atlassian.net/browse/MOSIP-34370)

- [Accessibility_Report-Partner Management (2).xlsx](https://mosip.atlassian.net/browse/MOSIP-34370)

- [Accessibility_Report-Partner Management (3).xlsx](https://mosip.atlassian.net/browse/MOSIP-34370)

- [Accessibility_Report-Partner Management (4).xlsx](https://mosip.atlassian.net/browse/MOSIP-34370)

- [Accessibility_Report-Partner Management (5).xlsx](https://mosip.atlassian.net/browse/MOSIP-34370)

- [Accessibility_Report-Partner Management (6).xlsx](https://mosip.atlassian.net/browse/MOSIP-34370)

- [Accessibility_Report-Partner Management (7).xlsx](https://mosip.atlassian.net/browse/MOSIP-34370)

- [Accessibility_Report-Partner Management (8).xlsx](https://mosip.atlassian.net/browse/MOSIP-34370)

- [Accessibility_Report-Partner Management (9).xlsx](https://mosip.atlassian.net/browse/MOSIP-34370)

**Comment by Ivanmeneges** [comment]
Mayura Deshmukh commented: Is IBM Equal Access Accessibility Checker test in scope [~accountid:712020:5a1aca27-bb82-4507-99ee-c8ab6211b59f] ?

**Comment by Ivanmeneges** [comment]
M Vibha Menon commented: [~accountid:636a272c11c69c7418450dbe] [~accountid:61482f457a3c8800672137a3] - can you help us understand about IBM Equal Access Accessibility Checker. This was never a part of our requirement/scope and none of us are aware of it

**Comment by Ivanmeneges** [comment]
Ragini Krishna commented: [~accountid:557058:694ce5f1-ed1b-41f7-a96a-879cb8a7e6c6] As discussed, we have executed the run using accessibility plug ins, discrepancies have been raised as Jira tickets. please take a look. also, can you please clarify [~accountid:712020:5a1aca27-bb82-4507-99ee-c8ab6211b59f]'s  queries.

**Comment by Ivanmeneges** [comment]
Mayura Deshmukh commented: As discussed with [~accountid:557058:694ce5f1-ed1b-41f7-a96a-879cb8a7e6c6] post 1.3.0-dp.1 release, we will check the report and do analysis of the issues reported.

**Comment by Ivanmeneges** [comment]
swetha.N commented: [^Accessibility_Report-Partner Management (6).xlsx]
[^Accessibility_Report-Partner Management (7).xlsx]
[^Accessibility_Report-Partner Management (8).xlsx]
[^Accessibility_Report-Partner Management (9).xlsx]
[^Accessibility_Report-Partner Management (10).xlsx]
[^Accessibility_Report-Partner Management (11).xlsx]
[^Accessibility_Report-Partner Management (12).xlsx]
[^Accessibility_Report-Partner Management (13).xlsx]
[^Accessibility_Report-Partner Management (14).xlsx]
